src/bdc/steps/enrich_custom_numbers.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Fabian-Paul Utech <f.utech@gmx.net>
+
+from typing import Optional
+
+
+import pandas as pd
+import phonenumbers
+from phonenumbers import geocoder
+
+
+from bdc.steps.step import Step
+
+
+class EnrichCustomNumbers(Step):
```

Comment:
Maybe rename to `PreprocessPhoneNumbers` or `ValidatePhoneNumbers`? The "Custom" keyword was more specific to the email-domain.

Reply:
PreprocessPhonenumbers it is